_layouts/home.html

```diff
@@ -3,114 +3,16 @@
 permalink: /
 ---
 
-<header id="top" class="hero" class="color-grid">
-  <div class="container text-white">
-      <div class="row d-flex align-items-center">
-          <div class="col-md-4">
-              <h1 class="choose-logo mb-4"><img src="/assets/img/logo-b4.svg" alt="{{ page.title }}" /></h1>
-          </div>
-          <div class="col-md-8">
-              {{ content }}
-          </div>
-      </div>
-  </div>
-</header>
+{% include page-types/hero.html  %}
 
-<section id="grid" class="scroll-bg grid" data-color="partner">  
-  <div class="container scroll-bg">   
-      <div class="row g-4 mb-2 mb-md-4">
-          <div class="col-md-4">
-              {% include cards/grid-text.html 
-                  title="PRX reached a weekly audience of 6.8 million on air and online."
-                  image-url="img/2guys.jpg" 
-                  alt="two guys at a workshop" 
-                  hover-text="6.8 million" %}
-          </div>
-          <div class="col-md-4">
-              {% include cards/grid-text.html 
-                  title="PRX reaches over 28 million people on-air and online each month." 
-                  type="organization"
-                  hover-text="28,000,000" %}
-          </div>
-          <div class="col-md-4">
-              {% include cards/grid-text.html 
-                  title="PRX saw nearly 60 million monthly downloads across our distributed podcasts." 
-                  hover-text="60 million"
-                  image-url="img/laughing.jpg" 
-                  alt="two guys at a workshop"  %}
-          </div>
-      </div>
-  </div>
-</section>
+{% include page-types/grid.html  %}
 
 {% assign pages = site.pages | where_exp: "page", "page.type contains 'welcome'" %}
-{% for page in pages reversed %}
-<section id="{{ page.prx-id }}" class="scroll-bg first {{ page.prx-path }}" data-color="{{ page.prx-path }}">
-  <div class="container">
-    <h2 class="page-heading">{{ page.title }}</h2>
-    {{ page.content }}
-  </div>
-
-  <aside class="prx-choose container">
-    <div class="arrow-container top">
-      <div class="arrow-start"></div>
-      <div class="arrow-center"></div>
-      <div class="arrow-end"></div>
-    </div>
-
-    <div class="row g-4">
-      <div class="col-md-4">
-        {% include cards/partner-card.html  %}
-      </div>
-      <div class="col-md-4">
-        {% include cards/creator-card.html  %}
-      </div>
-      <div class="col-md-4">
-        {% include cards/listener-card.html  %}
-      </div>
-    </div>   
-    
-    <div class="arrow-container bottom">
-      <div class="arrow-start"></div>
-      <div class="arrow-center"></div>
-      <div class="arrow-end"></div>
-    </div>
-  </aside>
-</section>
+{% for page in pages %}
+  {% include page-types/welcome.html  %}
 {% endfor %}
 
 {% assign pages = site.pages | where_exp: "page", "page.type contains 'section'" %}
 {% for page in pages reversed %}
```

Comment:
How can I set this so the variable `order` determines the sort order? Right now i think its sorting by date created.

Reply:
`{% assign pages = site.pages | where_exp: "page", "page.type contains 'section'" | sort: "order" %}`

Reply:
That worked well. thank you!!!